modules/bazel_sonarqube/1.0.1/presubmit.yml

```diff
@@ -0,0 +1,18 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+    - '@bazel_sonarqubesonarqube'
```

Comment:
This looks like a typo

Reply:
Oh thanks for catching that. I will need to update the original repo first to add a target to test.

Reply:
@fmeum I believe this is now ready to go.